pkg/odo/cli/utils/config.go

```diff
@@ -22,7 +22,8 @@ var configurationCmd = &cobra.Command{
 Available Parameters:
 UpdateNotification - Controls if an update notification is shown or not (true or false)
 NamePrefix - Default prefix is the current directory name. Use this value to set a default name prefix
-Timeout - Timeout (in seconds) for OpenShift server connection check`,
+Timeout - Timeout (in seconds) for OpenShift server connection check
+PodTimeout - Pod Timeout (in seconds) to wait for an OpenShift Pod to start`,
```

Comment:
odo user might not know when this will happen, or even what is a pod.

This should be explained in odo terms or commands that will this effect.